config/crd/bases/operator.open-cluster-management.io_multiclusterhubs.yaml

```
@@ -194,6 +194,11 @@ spec:
                 required:
                 - failedProvisionConfig
                 type: object
+              hubSize:
+                description: The resource allocation bucket for this hub to use.
+                  [S (Small), M (Medium), L (Large), XL (Extra Large)]. Defaults to
+                  (M)edium if not specified.
+                type: ["S", "M", "L", "XL"]
```

For the hub size, I believe we can just use `type: string` to define the size of the resources:
```suggestion
              hubSize:
                description: The resource allocation bucket for this hub to use.
                  [S (Small), M (Medium), L (Large), XL (Extra Large)]. Defaults to
                  (M)edium if not specified.
                type: string
                enum:
                  - S
                  - M
                  - L
                  - XL
                default: M
```